src/utils/transactions.js

```diff
@@ -182,11 +186,11 @@ const mapTransactionToTransactionPayload = (transaction: EthereumTransaction): T
 };
 
 export const getGasDecimals = (chain: Chain, gasToken: ?GasToken) => {
-  return gasToken?.decimals ?? nativeAssetPerChain[chain].decimals;
+  return gasToken?.decimals ?? nativeAssetPerChain[chain].decimals ?? 18;
```

Comment:
Seems like this change could actually mask the real problem, which is passing `null/undefined` chain object, which should be handled at call site(s). wdyt?

Reply:
Hm, would make sense, but then I would suggest expand these gas calls with a check if `nativeAssetPerChain[chain]` exists for `chain` and log it if it doesn't, wdyt?

Reply:
yeah, we could add an error log here

Reply:
Added.